backend-api/src/functions/asyncToken/asyncTokenHandler.ts

```diff
@@ -33,101 +33,103 @@ export async function lambdaHandlerConstructor(
   logger.addContext(context);
   logger.log("STARTED");
 
-  const configResponse = new ConfigService().getConfig(dependencies.env);
-  if (configResponse.isError) {
+  const configResult = new ConfigService().getConfig(dependencies.env);
+  if (configResult.isError) {
     logger.log("ENVIRONMENT_VARIABLE_MISSING", {
-      errorMessage: configResponse.value,
+      errorMessage: configResult.value,
     });
     return serverErrorResponse;
   }
 
-  const config = configResponse.value;
+  const config = configResult.value;
 
   // Ensure that request contains expected params
   const requestService = dependencies.requestService();
-  const processRequest = requestService.processRequest(event);
+  const processRequestResult = requestService.processRequest(event);
 
-  if (processRequest.isError) {
-    if (processRequest.value === "Invalid grant_type") {
-      logger.log("INVALID_REQUEST", { errorMessage: processRequest.value });
+  if (processRequestResult.isError) {
+    if (processRequestResult.value === "Invalid grant_type") {
+      logger.log("INVALID_REQUEST", {
+        errorMessage: processRequestResult.value,
+      });
       return badRequestResponseInvalidGrant;
     }
 
-    logger.log("INVALID_REQUEST", { errorMessage: processRequest.value });
+    logger.log("INVALID_REQUEST", { errorMessage: processRequestResult.value });
 
     return badRequestResponseInvalidAuthorizationHeader;
   }
 
-  const suppliedCredentials = processRequest.value;
+  const suppliedClientCredentials = processRequestResult.value;
 
   // Fetching stored client credentials
-  const clientCredentialsService = dependencies.clientCredentialService();
-  const clientCredentialsResult =
-    await clientCredentialsService.getClientCredentials();
-  if (clientCredentialsResult.isError) {
+  const clientCredentialsService = dependencies.clientCredentialsService();
+  const storedClientCredentialsArrayResult =
```

Comment:
I don't think we need Array in the name. I'm also not sure why we've introduced "stored"